### PR TITLE
 Change text in e2e UX to new copy

### PR DESCRIPTION
**Pull request by dbkr**

### CODE REVIEW

File: src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js [hunk 1]
```diff
@@ -265,8 +264,15 @@ export default React.createClass({
...
-            <p>{_t("You'll need it if you log out or lose access to this device.")}</p>
+            <p>{_t(
+                "<b>Warning</b>: you should only set up key backup from a trusted computer.", {},
+                { b: sub => <b>{sub}</b> },
+            )}</p>
+            <p>{_t(
+                "We'll store an encrypted copy of your keys on our server. " +
+                "Protect your backup with a passphrase to keep it secure.",
+            )}</p>
+            <p>{_t("For maximum security, this should be different from your account password.")}</p>
```
Comment (turt2live) [inline src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js]: recommendation: "This should be unique and not used anywhere else".

(I think @lampholder is responsible for the wording though?)

File: src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js [hunk 2]
```diff
@@ -353,9 +337,7 @@ export default React.createClass({
         const DialogButtons = sdk.getComponent('views.elements.DialogButtons');
         return <div>
             <p>{_t(
-                "Type in your Recovery Passphrase to confirm you remember it. " +
-                "If it helps, add it to your password manager or store it " +
-                "somewhere safe.",
+                "Please type in your passphrase again to check you remember it.",
```
Comment (turt2live) [inline src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js]: This feels like it is missing words, but still passes the "is English" test.

File: src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js [hunk 3]
```diff
@@ -484,19 +471,19 @@ export default React.createClass({
...
-                return _t('Create a Recovery Passphrase');
+                return _t('Secure your backup with a passphrase');
             case PHASE_PASSPHRASE_CONFIRM:
-                return _t('Confirm Recovery Passphrase');
+                return _t('Confirm your passphrase');
             case PHASE_OPTOUT_CONFIRM:
                 return _t('Warning!');
             case PHASE_SHOWKEY:
-                return _t('Recovery Key');
+                return _t('Recovery key');
```
Comment (turt2live) [inline src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js]: We capitalize `Key` elsewhere on buttons and  things - why lowercase it here?

File: src/components/views/rooms/RoomRecoveryReminder.js [hunk 4]
```diff
@@ -104,46 +109,54 @@ export default class RoomRecoveryReminder extends React.PureComponent {
...
+                    <p>{_t(
+                        "Messages in this room are secured with end-to-end " +
+                        "encryption. Only you and the recipient(s) have the " +
+                        "keys to read these messages.",
+                    )}</p>
+                    <p>{_t(
+                        "Securely back up your keys to avoid losing them. " +
+                        "<a>Learn more.</a>", {},
+                        {
+                            // We don't have this link yet: this will prevent the translators
```
Comment (turt2live) [inline src/components/views/rooms/RoomRecoveryReminder.js]: TODO please so it shows up in various trackers (although I might be the only person that uses these panels...)

File: src/components/views/settings/KeyBackupPanel.js [hunk 5]
```diff
@@ -146,13 +150,35 @@ export default class KeyBackupPanel extends React.PureComponent {
             return <Spinner />;
         } else if (this.state.backupInfo) {
             let clientBackupStatus;
+            let buttons;
+
             if (MatrixClientPeg.get().getKeyBackupEnabled()) {
-                clientBackupStatus = _t("This device is using key backup");
+                clientBackupStatus = <div>
+                    <p>{encryptedMessageAreEncrypted}</p>
+                    <p>{_t("This device is backing up your keys. ✅")}</p>
```
Comment (turt2live) [inline src/components/views/settings/KeyBackupPanel.js]: somewhat of an academic question: should the emoji be translatable? are we aware of any languages where the placement of said emoji will affect the sentence structure/message? 

also should we be using emojione for consistency?

File: src/components/views/settings/KeyBackupPanel.js [hunk 6]
```diff
@@ -146,13 +150,35 @@ export default class KeyBackupPanel extends React.PureComponent {
...
             } else {
-                clientBackupStatus = _t(
-                    "This device is <b>not</b> using key backup. Restore the backup to start using it.", {},
-                    {b: x => <b>{x}</b>},
-                );
+                clientBackupStatus = <div>
+                    <p>{encryptedMessageAreEncrypted}</p>
+                    <p>{_t(
+                        "This device is <b>not backing up your keys</b>.", {},
+                        {b: x => <b>{x}</b>},
```
Comment (turt2live) [inline src/components/views/settings/KeyBackupPanel.js]: nit: we use `sub` elsewhere instead of `x`

File: src/components/views/settings/KeyBackupPanel.js [hunk 7]
```diff
@@ -241,20 +267,20 @@ export default class KeyBackupPanel extends React.PureComponent {
...
-                </p>
+                {buttons}
             </div>;
         } else {
             return <div>
-                {_t("No backup is present")}<br /><br />
+                <div>
+                    <p>{_t(
+                        "Your keys are <b>not being backed up from this device</b>.", {},
+                        {b: x => <b>{x}</b>},
```
Comment (turt2live) [inline src/components/views/settings/KeyBackupPanel.js]: `sub`

File: src/i18n/strings/en_EN.json [hunk 8]
```diff
@@ -1509,15 +1510,16 @@
     "<b>Print it</b> and store it somewhere safe": "<b>Print it</b> and store it somewhere safe",
     "<b>Save it</b> on a USB key or backup drive": "<b>Save it</b> on a USB key or backup drive",
     "<b>Copy it</b> to your personal cloud storage": "<b>Copy it</b> to your personal cloud storage",
-    "Your encryption keys are now being backed up in the background to your Homeserver. The initial backup could take several minutes. You can view key backup upload progress in Settings.": "Your encryption keys are now being backed up in the background to your Homeserver. The initial backup could take several minutes. You can view key backup upload progress in Settings.",
+    "Your keys are being backed up (the first backup could take a few minutes).": "Your keys are being backed up (the first backup could take a few minutes).",
+    "Okay": "Okay",
```
Comment (turt2live) [inline src/i18n/strings/en_EN.json]: fwiw this is somewhat indicative of an inconsistency somewhere - we almost certainly have an `OK` somewhere, but we're using a different one here. Probably not worth fixing at this point though.

**Comment by turt2live** [comment]
Also your build passed but github lost the status update.

File: src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js (hunk 2 above)
Comment (dbkr) [reply src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js]: 'type your passphrase in again' may be better?

File: src/components/views/settings/KeyBackupPanel.js (hunk 5 above)
Comment (dbkr) [reply src/components/views/settings/KeyBackupPanel.js]: Very good point - fixed

File: src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js (hunk 2 above)
Comment (turt2live) [reply src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js]: It falls apart for me at "to check you remember it" - it feels like some context is missing at least. Something like "Type in your passphrase again to ensure it is correct" might work?

**Comment by dbkr** [comment]
Have addressed a bunch of the comments - rest of these for @lampholder I think

File: src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js (hunk 1 above)
Comment (lampholder) [reply src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js]: I like the suggestion, but am afraid its broader truth is missing the specific critical point (and at the risk of overthinking it, people might be confused by 'not used anywhere else' - _is_ somewhere else in Riot 'somewhere else'?).

I can't immediately think how to adopt the spirit of the suggestion without losing that specific key point (or making it too wordy) so I'm inclined to stick with the original at the moment :S

File: src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js (hunk 2 above)
Comment (lampholder) [reply src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js]: Agreed, this is stumbly.

The 'to check you remember it' business taking a leaf out of Monzo's book and trying to speak like a human (rather than the more robotic alternative: "ENTER YOUR PASSPHRASE TO CONFIRM").

I think 1password says something warm and friendly like "Looks good! Let's try practicing that one more time to make sure you know it off by heart!" but I'm afraid of that getting lost in (multiple) translations. 

You're right, though - we're actually not just doing this to check they remember it - we're doing it to minimise the risk of their having typooed it the first time.

How about: "Please enter your passphrase a second time to confirm"?

File: src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js (hunk 3 above)
Comment (lampholder) [reply src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js]: We've been treating Key Backup as a proper noun, and recovery key as a common noun.

Might be worth reflecting on when we've got a bit more breathing space, but I'm happy with that decision for now :)

Comment (lampholder) [reply src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js]: Where 'happy with that decision' == leave it how it is I mean.

File: src/i18n/strings/en_EN.json (hunk 8 above)
Comment (lampholder) [reply src/i18n/strings/en_EN.json]: Bleaugh, you're probably right. One to stomp out on another day, though (unless it's easy to refer to the translation files to identify whether we should Okay or OK)

File: src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js (hunk 2 above)
Comment (dbkr) [reply src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js]: Done
